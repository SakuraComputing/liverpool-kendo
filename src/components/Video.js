import React from 'react';
import ReactPlayer from 'react-player';

class Video extends React.Component {
    render () {
        return (
            <div className="content-container">
                <h2>Japanology Kendo</h2>
                <div className="player-wrapper">
                    <ReactPlayer
                        className='react-player'
                        url='https://www.youtube.com/watch?v=8Ea2ih7uNW4'
                        width='100%'
                        height='100%'
                        controls={true}
                    />
                </div>
                <div>30 minute programme about kendo and what you should expect to get from it.</div>
                <h2>Mariko Yamamoto - Women's Kendo Champion</h2>
                <div className="player-wrapper">
                    <ReactPlayer
                        className='react-player'
                        url='https://www.youtube.com/watch?v=D2Vei2tMgp4'
                        width='100%'
                        height='100%'
                        controls={true}
                    />
                </div>
                <div>Kendo shows your heart.</div>
                <h2>Kendo 8th Dan challenge the toughest exam in Japan</h2>
                <div className="player-wrapper">
                    <ReactPlayer
                        className="react-player"
                        url="https://www.youtube.com/watch?v=My-XxYovtkY"
                        width='100%'
                        height='100%'
                        controls={true}
                    />
                </div>
                <div>The pass rate is around 0.5%</div>
                <h2>Kendo Kata</h2>
                <div className="player-wrapper">
                    <ReactPlayer
                        className="react-player"
                        url="https://www.youtube.com/watch?v=9UfLrmDk8UE&list=PLaKZ0PAEUfwVTvUbLoUQh3pSxChpqOXpM"
                        width='100%'
                        height='100%'
                        controls={true}
                    />
                </div>
                <div>Kendo kata are vitally important as part of kendo study</div>
                <h2>Mochida Sensei 10th Dan</h2>
                <div className="player-wrapper">
                    <ReactPlayer
                        className='react-player'
                        url='https://www.youtube.com/watch?v=_XHSZ-sLG3I&t=6s'
                        width='100%'
                        height='100%'
                        controls={true}
                    />
                </div>
                <div>Kendo is a lifelong study one in which experience is considered more important than physique</div>
            </div>
        )
    }
}
export default Video;
