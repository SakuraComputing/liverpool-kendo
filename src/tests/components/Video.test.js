import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Video from '../../components/Video';

test('should render the Video page',() => {
    const wrapper = shallow(<Video />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});
