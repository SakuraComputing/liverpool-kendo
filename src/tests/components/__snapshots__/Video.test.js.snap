// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Video page 1`] = `
<div
  className="content-container"
>
  <h2>
    Japanology Kendo
  </h2>
  <div
    className="player-wrapper"
  >
    <ReactPlayer
      className="react-player"
      config={
        Object {
          "dailymotion": Object {
            "params": Object {
              "api": 1,
              "endscreen-enable": false,
            },
            "preload": false,
          },
          "facebook": Object {
            "appId": "1309697205772819",
          },
          "file": Object {
            "attributes": Object {},
            "forceAudio": false,
            "forceDASH": false,
            "forceHLS": false,
            "hlsOptions": Object {},
            "tracks": Array [],
          },
          "soundcloud": Object {
            "options": Object {
              "buying": false,
              "download": false,
              "liking": false,
              "sharing": false,
              "show_comments": false,
              "show_playcount": false,
              "visual": true,
            },
          },
          "vimeo": Object {
            "playerOptions": Object {
              "autopause": false,
              "autoplay": false,
              "byline": false,
              "portrait": false,
              "title": false,
            },
            "preload": false,
          },
          "wistia": Object {
            "options": Object {},
          },
          "youtube": Object {
            "playerVars": Object {
              "autoplay": 0,
              "iv_load_policy": 3,
              "modestbranding": 1,
              "playsinline": 1,
              "rel": 0,
              "showinfo": 0,
            },
            "preload": false,
          },
        }
      }
      controls={true}
      height="100%"
      loop={false}
      muted={false}
      onBuffer={[Function]}
      onDuration={[Function]}
      onEnded={[Function]}
      onError={[Function]}
      onPause={[Function]}
      onPlay={[Function]}
      onProgress={[Function]}
      onReady={[Function]}
      onSeek={[Function]}
      onStart={[Function]}
      playbackRate={1}
      playing={false}
      playsinline={false}
      progressInterval={1000}
      style={Object {}}
      url="https://www.youtube.com/watch?v=8Ea2ih7uNW4"
      volume={0.8}
      width="100%"
    />
  </div>
  <div>
    30 minute programme about kendo and what you should expect to get from it.
  </div>
  <h2>
    Mariko Yamamoto - Women's Kendo Champion
  </h2>
  <div
    className="player-wrapper"
  >
    <ReactPlayer
      className="react-player"
      config={
        Object {
          "dailymotion": Object {
            "params": Object {
              "api": 1,
              "endscreen-enable": false,
            },
            "preload": false,
          },
          "facebook": Object {
            "appId": "1309697205772819",
          },
          "file": Object {
            "attributes": Object {},
            "forceAudio": false,
            "forceDASH": false,
            "forceHLS": false,
            "hlsOptions": Object {},
            "tracks": Array [],
          },
          "soundcloud": Object {
            "options": Object {
              "buying": false,
              "download": false,
              "liking": false,
              "sharing": false,
              "show_comments": false,
              "show_playcount": false,
              "visual": true,
            },
          },
          "vimeo": Object {
            "playerOptions": Object {
              "autopause": false,
              "autoplay": false,
              "byline": false,
              "portrait": false,
              "title": false,
            },
            "preload": false,
          },
          "wistia": Object {
            "options": Object {},
          },
          "youtube": Object {
            "playerVars": Object {
              "autoplay": 0,
              "iv_load_policy": 3,
              "modestbranding": 1,
              "playsinline": 1,
              "rel": 0,
              "showinfo": 0,
            },
            "preload": false,
          },
        }
      }
      controls={true}
      height="100%"
      loop={false}
      muted={false}
      onBuffer={[Function]}
      onDuration={[Function]}
      onEnded={[Function]}
      onError={[Function]}
      onPause={[Function]}
      onPlay={[Function]}
      onProgress={[Function]}
      onReady={[Function]}
      onSeek={[Function]}
      onStart={[Function]}
      playbackRate={1}
      playing={false}
      playsinline={false}
      progressInterval={1000}
      style={Object {}}
      url="https://www.youtube.com/watch?v=D2Vei2tMgp4"
      volume={0.8}
      width="100%"
    />
  </div>
  <div>
    Kendo shows your heart.
  </div>
  <h2>
    Kendo 8th Dan challenge the toughest exam in Japan
  </h2>
  <div
    className="player-wrapper"
  >
    <ReactPlayer
      className="react-player"
      config={
        Object {
          "dailymotion": Object {
            "params": Object {
              "api": 1,
              "endscreen-enable": false,
            },
            "preload": false,
          },
          "facebook": Object {
            "appId": "1309697205772819",
          },
          "file": Object {
            "attributes": Object {},
            "forceAudio": false,
            "forceDASH": false,
            "forceHLS": false,
            "hlsOptions": Object {},
            "tracks": Array [],
          },
          "soundcloud": Object {
            "options": Object {
              "buying": false,
              "download": false,
              "liking": false,
              "sharing": false,
              "show_comments": false,
              "show_playcount": false,
              "visual": true,
            },
          },
          "vimeo": Object {
            "playerOptions": Object {
              "autopause": false,
              "autoplay": false,
              "byline": false,
              "portrait": false,
              "title": false,
            },
            "preload": false,
          },
          "wistia": Object {
            "options": Object {},
          },
          "youtube": Object {
            "playerVars": Object {
              "autoplay": 0,
              "iv_load_policy": 3,
              "modestbranding": 1,
              "playsinline": 1,
              "rel": 0,
              "showinfo": 0,
            },
            "preload": false,
          },
        }
      }
      controls={true}
      height="100%"
      loop={false}
      muted={false}
      onBuffer={[Function]}
      onDuration={[Function]}
      onEnded={[Function]}
      onError={[Function]}
      onPause={[Function]}
      onPlay={[Function]}
      onProgress={[Function]}
      onReady={[Function]}
      onSeek={[Function]}
      onStart={[Function]}
      playbackRate={1}
      playing={false}
      playsinline={false}
      progressInterval={1000}
      style={Object {}}
      url="https://www.youtube.com/watch?v=My-XxYovtkY"
      volume={0.8}
      width="100%"
    />
  </div>
  <div>
    The pass rate is around 0.5%
  </div>
  <h2>
    Kendo Kata
  </h2>
  <div
    className="player-wrapper"
  >
    <ReactPlayer
      className="react-player"
      config={
        Object {
          "dailymotion": Object {
            "params": Object {
              "api": 1,
              "endscreen-enable": false,
            },
            "preload": false,
          },
          "facebook": Object {
            "appId": "1309697205772819",
          },
          "file": Object {
            "attributes": Object {},
            "forceAudio": false,
            "forceDASH": false,
            "forceHLS": false,
            "hlsOptions": Object {},
            "tracks": Array [],
          },
          "soundcloud": Object {
            "options": Object {
              "buying": false,
              "download": false,
              "liking": false,
              "sharing": false,
              "show_comments": false,
              "show_playcount": false,
              "visual": true,
            },
          },
          "vimeo": Object {
            "playerOptions": Object {
              "autopause": false,
              "autoplay": false,
              "byline": false,
              "portrait": false,
              "title": false,
            },
            "preload": false,
          },
          "wistia": Object {
            "options": Object {},
          },
          "youtube": Object {
            "playerVars": Object {
              "autoplay": 0,
              "iv_load_policy": 3,
              "modestbranding": 1,
              "playsinline": 1,
              "rel": 0,
              "showinfo": 0,
            },
            "preload": false,
          },
        }
      }
      controls={true}
      height="100%"
      loop={false}
      muted={false}
      onBuffer={[Function]}
      onDuration={[Function]}
      onEnded={[Function]}
      onError={[Function]}
      onPause={[Function]}
      onPlay={[Function]}
      onProgress={[Function]}
      onReady={[Function]}
      onSeek={[Function]}
      onStart={[Function]}
      playbackRate={1}
      playing={false}
      playsinline={false}
      progressInterval={1000}
      style={Object {}}
      url="https://www.youtube.com/watch?v=9UfLrmDk8UE&list=PLaKZ0PAEUfwVTvUbLoUQh3pSxChpqOXpM"
      volume={0.8}
      width="100%"
    />
  </div>
  <div>
    Kendo kata are vitally important as part of kendo study
  </div>
  <h2>
    Mochida Sensei 10th Dan
  </h2>
  <div
    className="player-wrapper"
  >
    <ReactPlayer
      className="react-player"
      config={
        Object {
          "dailymotion": Object {
            "params": Object {
              "api": 1,
              "endscreen-enable": false,
            },
            "preload": false,
          },
          "facebook": Object {
            "appId": "1309697205772819",
          },
          "file": Object {
            "attributes": Object {},
            "forceAudio": false,
            "forceDASH": false,
            "forceHLS": false,
            "hlsOptions": Object {},
            "tracks": Array [],
          },
          "soundcloud": Object {
            "options": Object {
              "buying": false,
              "download": false,
              "liking": false,
              "sharing": false,
              "show_comments": false,
              "show_playcount": false,
              "visual": true,
            },
          },
          "vimeo": Object {
            "playerOptions": Object {
              "autopause": false,
              "autoplay": false,
              "byline": false,
              "portrait": false,
              "title": false,
            },
            "preload": false,
          },
          "wistia": Object {
            "options": Object {},
          },
          "youtube": Object {
            "playerVars": Object {
              "autoplay": 0,
              "iv_load_policy": 3,
              "modestbranding": 1,
              "playsinline": 1,
              "rel": 0,
              "showinfo": 0,
            },
            "preload": false,
          },
        }
      }
      controls={true}
      height="100%"
      loop={false}
      muted={false}
      onBuffer={[Function]}
      onDuration={[Function]}
      onEnded={[Function]}
      onError={[Function]}
      onPause={[Function]}
      onPlay={[Function]}
      onProgress={[Function]}
      onReady={[Function]}
      onSeek={[Function]}
      onStart={[Function]}
      playbackRate={1}
      playing={false}
      playsinline={false}
      progressInterval={1000}
      style={Object {}}
      url="https://www.youtube.com/watch?v=_XHSZ-sLG3I&t=6s"
      volume={0.8}
      width="100%"
    />
  </div>
  <div>
    Kendo is a lifelong study one in which experience is considered more important than physique
  </div>
</div>
`;
